src/validator.js

```diff
@@ -36,6 +37,7 @@ export default class Validator {
     this.chalk = new chalk.constructor(
       {enabled: !this.config.boring});
     this.collector = new Collector();
+    this.addonMetaData = {};
```

Comment:
Actually, it involves extra tests and such so I'll make a new issue for it, just to keep this lean. #257